frontend/src/components/pages/ViewDonations.tsx

```diff
@@ -16,12 +16,17 @@ import SchedulingAPIClient from "../../APIClients/SchedulingAPIClient";
 import { Schedule } from "../../types/SchedulingTypes";
 import Calendar from "../common/Calendar/Calendar";
 
-const ViewDonations = (): React.ReactElement => {
+const ViewDonations = ({
+  isPublicView,
+}: {
+  isPublicView?: boolean;

```

Comment:
what is this isPublicView boolean??

Reply:
![image](https://user-images.githubusercontent.com/43387820/145326459-343c5c23-c80d-43ab-a2ae-7e40fd98ddfe.png)
looks like this was a part of a "add view donations" PR and somehow got rolled into this one's diff files